src/update.js

```diff
@@ -0,0 +1,3 @@
+export default function setAsCompleted(todo) {
+  todo.completed = !todo.completed;
+}
```

Comment:
Kindly there is no need to create a hole module here just to include one little function.
You can create a utils.js and include all functions deals with tasks like Add, Remove, Edit, and remove completed tasks then import this module to index.js